css/logo-nav.css

```diff
@@ -56,100 +56,250 @@ animation-delay: 0.7s;
+
+    .nav>li>a {
+        font-size: 13px;
+        letter-spacing: 1px;
     }
-}
-@media(max-width:992px) {
-    .language-set #feature {
-        display: block;
+
+    .nav > li > a:hover {
+       background-color: #E7E7E7;
+       color: black;
     }
-    .notebook-preview img {
-        margin: 0 auto;
+
+    .nav > li > a:focus {
+        background-color: #E7E7E7 !important;
     }
-    #notebookicon {
+    .dropdown-menu>li>a {
+        font-size: 13px;
+        letter-spacing: 1px;
+    }
+    .dropdown-menu > li  {
+       background-color: #f6f6f6;
+    }
+    .dropdown-menu {
+        display: none !important;
+    }
+    .dropdown  a  {
+        text-align: left !important;
+    }
+    .dropdown  a  {
+        padding: 10px 15px 10px 15px !important;
+    }
+    .dropdown {
+        padding-left: 25px;
+    }
+    .navbar-toggle {
+        margin: 23px 0 23px 0;
+        padding: 9px 0 9px 0;
+    }
+    .navbar-toggle {
         display: block;
-        margin: 0 auto;
+        margin-left: 15px;
+    }
+    .navbar-header {
+        float: none;
+    }
+    .navbar-collapse {
+        border-top: 1px solid transparent;
+        box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
+    }
+    .navbar-fixed-top {
+        top: 0;
+        border-width: 0 0 1px;
+    }
+    .navbar-nav {
+        float: none!important;
+        margin-top: 7.5px;
+    }
+    .navbar-nav>li {
+        float: none;
+    }
+    .navbar-nav>li>a {
+        padding-top: 10px;
         padding-bottom: 10px;
     }
-    .notebook-preview h1 {
-        text-align: center;
+    .nav a {
+        width: auto;
+    }
+   .navbar-nav .open .dropdown-menu {
+       position: static;
+       float: none;
+       width: auto;
+       background-color: transparent;
+       border: 0;
+       -webkit-box-shadow: none;
+       box-shadow: none;
+    } 
+    .nav:first-child {
+        border-top: 2px solid #F2F2F2;
+    }
+    .nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
+        border-color: #F2F2F2;
+    }
+   .navbar-scroll .navbar-inner .dropdown-menu > :first-child {
+        border-top: none;
+    }
+    .open .dropdown-menu {
+        display: block !important;
+    }
+    .collapse.in{
+        display:block !important;
+    }
+    #maintabs a {
+        text-align: left !important;
+    }
+    #maintabs a {
+        padding: 10px 15px 10px 15px;
+    }
+    #carrot {
+        float: right;
+        padding-top: 8px;
+    }
+}
+@media (max-width: 768px) {
+    .navbar-header {
+        margin-left: 0px !important;
+        margin-right: 0px !important;
+    }
+    .dropdown  a  {
+        padding: 10px 15px 10px 15px !important;
+
+    }
+    #maintabs a {
+        padding: 10px 15px 10px 15px;
     }
 }
+
+/*------------------------------------------------*/
+
+.img-container {
+    position: absolute;
+    width: 95%;
+    height: 100%;
+    text-align: center;
+    margin: 0 auto;
+
+}
+.img-container img {
+    vertical-align: center;
```

Comment:
`vertical-align:` , `center`  is [not a valid value](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align)